pytest_fixtures/component/oscap.py

```diff
@@ -9,15 +9,14 @@
 from robottelo.config import settings
 from robottelo.constants import OSCAP_PROFILE
 from robottelo.constants import OSCAP_TAILORING_FILE
-from robottelo.helpers import get_data_file
 
 
 @pytest.fixture(scope="session")
 def tailoring_file_path(session_target_sat):
     """Return Tailoring file path."""
-    local = get_data_file(OSCAP_TAILORING_FILE)
+    local = PurePath('tests/foreman/data').joinpath(OSCAP_TAILORING_FILE)
```

Comment:
I don't think this change is a good one. There are now 46 individual direct references to this file path.

At least defining the path instance as an importable variable would be acceptable, but I don't think this helper is necessarily one that needs to go away.

There's a need/want to separate the file name itself from the local path, so that the file can be referenced on the remote server too.  This can be done with the properties of the Path instance, so maybe creating a Box class with file instances for the available data files would be a good way to directly reference files via a single import?

Reply:
Yeah, Even I felt bit weird for this. But now as per your comment, I could create class inheriting `Box` class in `robottelo/constants.py` whereas all the file paths could be instances of it. Something like :

```
class DataFile(Box):
    DATA_DIR = PurePath('tests/foreman/data')
    OSCAP_TAILORING_FILE = DATA_DIR.joinpath(OSCAP_TAILORING_FILE)
```

Access as:
```
from robottelo.constants import DataFile
local = DataFile.OSCAP_TAILORING_FILE
```
??

Reply:
Yep, exactly the kind of thing I was thinking of.  Callers don't have to import the whole module, and there's a singular point to list/locate data files consistently as PurePath objects.

Reply:
Done !